updater/spec/dependabot/experimental_grouped_updater_spec.rb

```diff
@@ -0,0 +1,1986 @@
+# frozen_string_literal: true
+
+# FIXME: This file is a copy-paste of `spec/dependabot/updater_spec.rb`
+#
+#       The intent is to run the tests for existing behaviour without the churn
+#       of refactoring the existing tests into shared examples until we know
+#       how the code is going to decompose
+#
+#       Tests that don't apply or work properly for grouped updates are skipped
+#       rather than deleted, and new group-specific tests are added under the
+#       `run_grouped` block.
```

Comment:
As noted, this test is a fork of the existing tests as just acts as a plumb-line for how much coverage we have on the new code, what is currently side-stepped, etc as a way to heatmap where we need to bring up coverage before we start changing the 'real' implementation.